pyserializer/fields.py

```diff
@@ -58,11 +60,24 @@ def __init__(self,
         :param validators: List of validators that should be ran when
             deserializing the field. This list will be appended along with
             the default_validators defined on each field.
+        :param error_messages: (dict) Custom error message on the field.
         """
         self.source = source
         self.label = label
         self.help_text = help_text
         self.validators = self.default_validators + (validators or [])
+        self.error_dict = OrderedDict()
+        if error_messages:
+            self.error_dict['type_name'] = kwargs.pop(
+                'type_name',
+                self.type_name
+            )
+            self.error_dict['type_label'] = kwargs.pop(
+                'type_label',
+                self.type_label
+            )
+            self.error_dict['message'] = error_messages['invalid']
```

Comment:
I think I prefer option 1, since thats a pattern I have seen in Dango Rest Framework.